src/components/Modals/Send/views/Address.tsx

```diff
@@ -28,6 +27,11 @@ import { AddressInput } from '../AddressInput/AddressInput'
 import { SendFormFields, SendInput } from '../Form'
 import { SendRoutes } from '../Send'
 
+const {
+  cosmos: { ChainAdapter: CosmosChainAdapter }
+} = cosmossdk
+const { ChainAdapter: EthereumChainAdapter } = ethereum
```

Comment:
Just a thought about alternative, since they are already being namespaced, could make sense to just reference them as such `ethereum.ChainAdapter` instead of the destructure and alias method.